### ISSUE TITLE
Support CMYK

### ISSUE THREAD

**Issue by liZe** [Kozea/WeasyPrint#1091]

**Comment by grewn0uille** [comment]
Hello!

As it’s soon our 2-year anniversary as CourtBouillon, we opened a [short survey](https://thym.courtbouillon.org/1) to know more about your expectations.
Don’t hesitate to support this feature and give it a boost 🚀!

The survey will be opened until October 10th.

**Comment by grewn0uille** [comment]
Hello!

As you may know, two weeks ago was [CourtBouillon 3-year anniversary](https://www.courtbouillon.org/blog/00046-three-years-of-courtbouillon) 🎉.

For this occasion, we prepared [a short survey](https://thym.courtbouillon.org/4) to have your opinion on this year’s features and to know what you’d like to see in the future!
Don’t hesitate to give a boost to this feature ✨️

 The survey is opened until November 19.

**Comment by zadeviggers** [comment]
Now that CSS colour 4 support is done, CSS colour 5 includes CMYK (to an extent): https://www.w3.org/TR/css-color-5/#device-cmyk

**Comment by liZe** [comment]
> Now that CSS colour 4 support is done, CSS colour 5 includes CMYK

Yes, we wrote [an article](https://www.courtbouillon.org/blog/00052-more-colors-in-weasyprint/) about colors in WeasyPrint 63.0.

Before adding CMYK support, it would be really useful to get feedback from Color 4 support. If anyone wants to test color spaces with 63.0, that would be really useful!

**Comment by grewn0uille** [comment]
WIP!

<img width="693" height="727" alt="Image" src="https://github.com/user-attachments/assets/586b8b79-24c4-42b5-9378-589d2056d2b5" />

**Comment by liZe** [comment]
Fixed by #2560.
